src/chapters/custom-auth.xml

```diff
@@ -403,5 +465,27 @@ public Map&lt;String, GuacamoleConfiguration>
 
 }</programlisting>
         </example>
+        <para>The extension is now complete and can be built as described earlier in <xref
+                xmlns:xlink="http://www.w3.org/1999/xlink" linkend="custom-auth-building"/>.</para>
+    </section>
+    <section xml:id="custom-auth-installing">
+        <title>Installing the extension</title>
+        <para>Guacamole extensions are self-contained <filename>.jar</filename> files which are
+            installed by being placed within <filename>GUACAMOLE_HOME/extensions</filename>, and
+            this extension is no different. As described in <xref
+                xmlns:xlink="http://www.w3.org/1999/xlink" linkend="configuring-guacamole"/>,
+                <varname>GUACAMOLE_HOME</varname> is a placeholder used to refer to the directory
+            that Guacamole uses to locate its configuration files and extensions. Typically, this
+            will be the <filename>.guacamole</filename> directory within the home directory of the
+            user running Tomcat.</para>
+        <para>To install your extension, ensure that the required properties have been added to your
+                <filename>guacamole.properties</filename>, copy the
+                <filename>target/guacamole-auth-tutorial-0.9.9.jar</filename> file into
+                <filename>GUACAMOLE_HOME/extensions</filename> and restart Tomcat. Guacamole will
+            automatically load your extension, logging an informative message that it has done
+            so:</para>
```

Comment:
It is indeed:

![yes-it-is-a-colon](https://cloud.githubusercontent.com/assets/4632905/11890291/7f95de82-a507-11e5-8a21-c24eef69adf1.png)